vmdb/app/models/ems_refresh/parsers/openstack_common/orchestration_stacks.rb

```diff
@@ -26,7 +27,7 @@ def detailed_stacks
           # TODO(lsmola) We need a support of GET /{tenant_id}/stacks/detail in FOG, it was implemented here
           # https://review.openstack.org/#/c/35034/, but never documented in API reference, so right now we
           # can't get list of detailed stacks in one API call.
-          @orchestration_service.stacks.collect(&:details)
+          @orchestration_service.stacks_for_accessible_tenants.collect(&:details)
```

Comment:
This comment is still valid. When the new API is ready (version and detail TBD), we need to refactor to call the new API for all accessible tenants.